### DISCUSSION TITLE
Missing Discussion voting

### DISCUSSION THREAD

**Discussion by TPS** [community/community#3496] [Mobile] [answered]
Related to https://github.com/github/feedback/discussions/3474, mobile app doesn't support discussion voting, only reactions.

[Requested screenshots](https://github.com/github/feedback/discussions/3496#discussioncomment-448589) of _this_ discussion from same device a few seconds apart:

GitHub Web:
![GitHub Web](https://user-images.githubusercontent.com/185902/110320153-e4ea6a80-7fdd-11eb-99c2-f8ee2a2eeddd.png)

GitHub App:
![GitHub App](https://user-images.githubusercontent.com/185902/110320188-ef0c6900-7fdd-11eb-8374-0aa3f141cd71.png)

**Comment by Shinigami92** [comment]
@TPS Just an idea, but could you add some screenshots as an example? :slightly_smiling_face:

**Comment by candyho** [comment]
Hey thanks for your feedback. Discussions voting is something on our radar, stay tuned!

**Comment by TPS** [comment, marked as answer]
&, apparently w/o notice or fanfare, this is implemented.  Well done! 🙇🏾‍♂️
